app/helpers/usage_helper.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+require 'plausible_api'
+module UsageHelper
+  def downloads(files)
+    files.map(&:downloads).inject(0, :+).to_s
+  end
+
+  def views
+    c = PlausibleApi::Client.new('pdc-discovery-staging.princeton.edu', (ENV['PLAUSIBLE_KEY'] || ''))
+    c.aggregate({ date: "2021-01-01,#{Time.zone.today.strftime('%Y-%m-%d')}", metrics: 'visitors,pageviews', filters: "event:page==/catalog/#{@document.id}" })["pageviews"]["value"]
+  rescue => e
+    logger.error "PLAUSIBLE ERROR: #{e.message}"
+    '0'
```

Comment:
Thanks!

Reply:
I think eventually we want to move code that interfaces with the Plausible API to its own Model/Service class to abstract Plausible from the views, but we can do that later.

Reply:
I think we should alert honeybadger when this error is raised. We're more likely to see it in honeybadger than we are in our logs. Doing so is easy: 

```
  Honeybadger.notify(exception)
```

See https://docs.honeybadger.io/lib/ruby/getting-started/reporting-errors/